e2e/__projects__/basic/test.js

```diff
@@ -118,8 +118,19 @@ test('supports relative paths when extending templates from .pug files', () => {
 })
 
 test('supports class component .vue files', () => {
-  mount(ClassComponent, { msg: 'Hello' })
-  expect(document.querySelector('div').textContent).toBe('Hello')
+  expect.assertions(3)
+  mount(ClassComponent, { msg: 'Props Message' })
+  expect(document.querySelector('[data-computed]').textContent).toBe(
+    'Message: Props Message'
+  )
+  expect(document.querySelector('[data-props]').textContent).toBe(
+    'Props Message'
+  )
+  const event = new window.Event('click')
+  document.querySelector('button').dispatchEvent(event)
+  nextTick().then(() => {
+    expect(document.querySelector('[data-methods]').textContent).toBe('Updated')
```

Comment:
Is this not a nicer way to get contents of an element?
```
wrapper.find('[data=methods]').text()
```

Reply:
See above